### ISSUE TITLE
[Bug]: WCO restore icon is off-center of the button when window is maximized

### ISSUE THREAD

**Issue by wheezard** [electron/electron#35038]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a bug report that matches the one I want to file, without success.

### Electron Version

19.0.8

### What operating system are you using?

Windows

### Operating System Version

Windows 10 21H1, build 19043.1826

### What arch are you using?

x64

### Last Known Working Electron version

19.0.0

### Expected Behavior

When window is maximized, the restore icon stays in center

### Actual Behavior

The restore icon appears to the top-left of the button center

### Testcase Gist URL

https://gist.github.com/wheezard/4882c46fa8ffb96f9a131bd6ae219c6e

### Additional Information

Unmaximized:
![Normal button](https://user-images.githubusercontent.com/90904039/180649362-30615c4c-4d3d-487b-a150-af38856e2932.png)

Maximized:
![Off-center button](https://user-images.githubusercontent.com/90904039/180649399-3976c402-0456-4703-a195-2ba12b149ac3.png)

**Comment by codebytere** [comment]
This is what i see running your repro:
<img width="1680" alt="Screen Shot 2022-07-25 at 9 55 01 AM" src="https://user-images.githubusercontent.com/2036040/180726855-b1346a71-c2df-456e-a152-3003a007f0c1.png">

which appears to be centered as expected.

**Comment by wheezard** [comment]
@codebytere Any success?

**Comment by codebytere** [comment]
No sadly - still centered for me 🤔

**Comment by wheezard** [comment]
Alright, I guess I wasn't running it in the right electron version, but with fiddle it also is off-center. This issue doesn't appear in v20.0.0, but in v20, the WCO just looks weird. Maybe that bug occurs only with some specific settings, I'll look into it later.

This is what I mean by "weird":
![screenshot](https://user-images.githubusercontent.com/90904039/182640528-d0d5e3fa-3180-4254-8cd7-4e67a699ea64.png)
![screenshot](https://user-images.githubusercontent.com/90904039/182640532-2054adeb-9596-4990-8c04-c1ff15535ee7.png)

**Comment by rzhao271** [comment]
I was looking into the misaligned restore button issue this week for https://github.com/microsoft/vscode/issues/157086.
I noticed the issue was fixed in v20.0.0-beta.9 and traced it back on GitHub to https://github.com/electron/electron/pull/34790. Therefore to fix this issue, I'd like to backport #34790 to Electron 19.

As for the additional spacing between buttons in v20, I'm thinking either it's because https://chromium.googlesource.com/chromium/src/+/30a32e6d6909e8e6e267b48bb7294e739b210c51%5E%21/#F0 wasn't added to Electron, or it's because of https://github.com/electron/electron/pull/34790/files#diff-2a87d5d8844989448e4571deacfd3f396b57cb5d9cdcac1d2b725e96ca57261fR64. For that, I'd like to create another PR that would need to be merged into the main branch and then backported to earlier versions.
